en/debug/porting-guide.md

```diff
@@ -2,55 +2,67 @@
-* Drivers: [DriverFramework](https://github.com/px4/DriverFramework).
-* Reference config: Running `make qurt_eagle_release` builds the Snapdragon Flight reference config.
+The configuration files for NuttX based boards, including linker scripts and other required settings are located under [platforms/nuttx/nuttx-configs](https://github.com/PX4/Firmware/tree/master/platforms/nuttx/nuttx-configs).
+
+The following example uses FMUv5 as it is the current [reference configuration](./reference-design.md) for NuttX based flight controllers:
+* Running `make px4fmu-v5_default` from the `src/Firmware` directory will build the FMUv5 config
+* The base FMUv5 configuration files are located in: [platforms/nuttx/nuttx-configs/px4fmu-v5](https://github.com/PX4/Firmware/tree/master/platforms/nuttx/nuttx-configs/px4fmu-v5).
+* Board specific header: [platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h](https://github.com/PX4/Firmware/blob/master/platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h).
+* NuttX OS config (created with Nuttx menuconfig): [nuttx-configs/px4fmu-v5/nsh/defconfig](https://github.com/PX4/Firmware/blob/master/platforms/nuttx/nuttx-configs/px4fmu-v5/nsh/defconfig).
+* Build configuration: [cmake/configs/nuttx\_px4fmu-v5\_default.cmake](https://github.com/PX4/Firmware/blob/master/cmake/configs/nuttx_px4fmu-v5_default.cmake).
 
+The function of each of these files, and perhaps more, will need to be duplicated for a new flight controller board.
 
-## Linux Boards
+#### NuttX Menuconfig
+If you need to modify the NuttX OS configuration, you can do this via [menuconfig](https://bitbucket.org/nuttx/nuttx) using the PX4 shortcuts:
+```sh
+make px4fmu-v5_default menuconfig
+make px4fmu-v5_default qconfig
+```
+
+### Linux
 
 Linux boards do not include the OS and kernel configuration. These are already provided by the Linux image available for the board (which needs to support the inertial sensors out of the box).
 
 * [cmake/configs/posix\_rpi\_cross.cmake](https://github.com/PX4/Firmware/blob/master/cmake/configs/posix_rpi_cross.cmake) - RPI cross-compilation.
 
+## Middleware Components and Configuration
 
+This section describes the various middleware components, and the configuration file updates needed to port them to new flight controller hardware.
 
+### QuRT / Hexagon
 
+* The start script is located in [posix-configs/](https://github.com/PX4/Firmware/tree/master/posix-configs).
+* The OS configuration is part of the default Linux image (TODO: Provide location of LINUX IMAGE and flash instructions).
+* The PX4 middleware configuration is located in [src/drivers/boards](https://github.com/PX4/Firmware/tree/master/src/drivers/boards). TODO: ADD BUS CONFIG
+* Drivers: [DriverFramework](https://github.com/px4/DriverFramework).
+* Reference config: Running `make qurt_eagle_release` builds the Snapdragon Flight reference config.
```

Comment:
This target isn't valid.

Reply:
@joshuawhitehead Can you fix this up too  - it should just be `make eagle_default` (according to [this](https://dev.px4.io/en/setup/building_px4.html#qurt--snapdragon-based-boards).

Reply:
Will do.  This was copy-paste from the current version of the porting guide, so the target may have changed since that was originally written.